code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -39,7 +39,7 @@
 
 	//Deathsquad space suit, not hardsuits because no flashlight!
```

Comment:
do we WANT them to not have flashlight, and have them carry the flashlight item in the suit? That sounds not very death-squad'dy

Reply:
I'm planning a Deathsquad balance PR after this to give them implants, hardsuit, etc.

Reply:
Also deathsquad currently have a Seclite in their bag, which will also be removed with hardsuits